client/app/email-domain/email-domain.service.js

```diff
@@ -730,14 +730,14 @@ angular.module("services").service(
 
         /**
          * Obtain Emails delegation
+         * @param {string|null} domain
          * @param {string|null} accountName
-         * @param {string} domain
          */
-        getDelegatedEmails (accountName, domain) {
+        getDelegatedEmails (domain = undefined, accountName = undefined) {
```

Comment:
If you don't pass arguments, domain / accountName are already undefined, so no need to define them with undefined value, right ?